### ISSUE TITLE
I am unable to run any of the samples 

### ISSUE THREAD

**Issue by ShamaUgale** [codeforequity-at/testmybot#101]
I am doing 
npm install 
npm run test

for example for slack this is the error i get
npm install
npm WARN testmybot-sample-slack@1.0.0 No repository field.

npm ERR! path /Users/shamau/Downloads/testmybot-master/samples/slack/node_modules/.staging/testmybot-23018b42/node_modules/accepts
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/Users/shamau/Downloads/testmybot-master/samples/slack/node_modules/.staging/testmybot-23018b42/node_modules/accepts' -> '/Users/shamau/Downloads/testmybot-master/samples/slack/node_modules/.staging/accepts-09f9c3ce'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/shamau/.npm/_logs/2018-10-31T14_40_24_408Z-debug.log

**Comment by codeforequity-at** [comment]
Please see the Troubleshooting guide in the Botium Wiki: https://github.com/codeforequity-at/botium-core/wiki/Troubleshooting

**Comment by ShamaUgale** [comment]
i did try all the options. i am still not able to run any of the samples here.

all i am doing is "npm install", is there any per-requisites needed? any specific version on node?

**Comment by codeforequity-at** [comment]
Did you have a look on the Botium CLI ? 
https://github.com/codeforequity-at/botium-core/wiki/The-Botium-CLI

Maybe this is more suitable for you. You don't need extra Node installation, everything is precompiled (for Windows, Linux, Mac).

**Comment by LacikIgor** [comment]
@ShamaUgale did u manage to work around this?

Experiencing the exact same issues - mac, node 8.10.0

**Comment by ShamaUgale** [comment]
No. I could not fix the issue. I am still facing it. By any chance were you able to figure out any solution to it? If i find one will document it and share across the community for sure.

**Comment by LacikIgor** [comment]
https://stackoverflow.com/questions/53519661/how-to-run-a-simple-botium-test-against-watson

This is how i ran botium and it worked

**Comment by ShamaUgale** [comment]
Thanks for the link @LacikIgor 
I will try this and update
